deploy/helm/metrics-collector-config.yaml

```diff
@@ -11,6 +11,55 @@ extensions:
 {{ toYaml .Values.otel.metrics.memory_ballast | indent 4 }}
 
 processors:
+{{- if .Values.otel.metrics.k8s_instrumentation.enabled }}
+  k8sattributes:
+    auth_type: "serviceAccount"
+    passthrough: false
+    extract:
+      metadata:
+        - k8s.deployment.name
```

Comment:
Didn't we want to add those k8s attributes always so we can create direct link between deployment and pod? In case it's done like this we will lose that with disabled instrumentation